test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -545,7 +553,18 @@ public void startup(ICluster cluster)
                 if (config.has(GOSSIP))
                 {
                     MigrationManager.setUptimeFn(() -> TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startedAt));
-                    StorageService.instance.initServer();
+                    try
+                    {
+                        StorageService.instance.initServer();
+                    }
+                    catch (Exception e)
+                    {
+                        // I am tired of looking up my notes for how to fix this... so why not tell the user?
```

Comment:
👍

Reply:
I was asked to move outside of this patch....